src/pages/dar_application/dar_application_new.css

```diff
@@ -49,5 +49,62 @@
   font-weight: 600;
   letter-spacing: 0;
   line-height: 24px;
+}
+
+/*///////////////////////////////////////////////////////COLLABORATOR*/
+
+.collaborator-summary-card {
```

Comment:
Optional feedback. The idea behind separating out CSS this way, as I understood it at the time, was intended to be component-specific. That would suggest this belong in a file called `Collaborator.css` or something similar.